src/core/js/views/notifyView.js

```diff
@@ -19,7 +19,8 @@ define([
         },
 
         setupEventListeners: function() {
-            this.listenTo(Adapt, 'remove', this.remove);
+            this.listenTo(Adapt, 'remove', this.closeNotify);
```

Comment:
Good call, I've done as you suggested Matt, as well as condensing remove and page:scrollTo to a single key